Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1351,6 +1351,20 @@ CanvasFiltersEnabled:
     WebCore:
       default: false
 
+CanvasNoiseInjectionEnabled:
```

Comment:
Is this separate feature flag needed? (Since this is already controlled by the same feature flag, which guards the presence of the hash salt).

Reply:
Adding a dedicated flag for this was a recommendation I got, given the current uncertainty around it.

Reply:
Okay, sounds good to me.